pkg/credentialprovider/keyring.go

```diff
@@ -111,6 +111,11 @@ func isDefaultRegistryMatch(image string) bool {
 		return true
```

Comment:
This is because an image reference to 'ubuntu' is resolved internally by docker to 'library/ubuntu', so the SplitN would be against just 'ubuntu'